app/src/main/res/values/strings.xml

```diff
@@ -285,19 +285,29 @@ Please report any problems you encounter via Github.</string>
     <string name="category_debug">Debug</string>
     <string name="category_experimental">Experimental</string>
 
-    <string name="always_run_in_background">Always run in background</string>
+    <!-- Preference screen - Run conditions -->
+    <string name="always_run_in_background">to_be_deleted</string>
+    <string name="always_run_in_background_enabled">to_be_deleted</string>
+    <string name="always_run_in_background_disabled">to_be_deleted</string>
+    <string name="sync_only_charging">to_be_deleted</string>
+    <string name="sync_only_wifi">to_be_deleted</string>
+    <string name="sync_only_wifi_ssids">to_be_deleted</string>
 
-    <!-- Preference summary in case it is enabled -->
-    <string name="always_run_in_background_enabled">Syncthing always runs in the background, according to preferences below.</string>
+    <string name="run_conditions_title">Run Conditions</string>
+    <string name="run_conditions_summary">Use the following options to decide when the syncthing UI will be available and synchronization takes place.</string>
```

Comment:
Use the following options to decide when Syncthing will run

Reply:
I put your suggestion in hoping no one gets confused that syncthing binary is not syncthing android app.